code/core/src/builder-manager/index.ts

```diff
@@ -157,7 +157,11 @@ const starter: StarterFunction = async function* starterGeneratorFn({
 
   yield;
 
-  const coreDirOrigin = join(dirname(require.resolve('@storybook/manager/package.json')), 'dist');
+  const coreDirOrigin = join(
```

Comment:
Shouldn't we use `storybook/core-path` here?

Reply:
`@storybook/core` cannot depend on `storybook` that would be a cyclical dependency.